cli/connect/output.go

```diff
@@ -0,0 +1,110 @@
+package connect
+
+import (
+	"fmt"
+
+	"github.com/tarantool/tt/cli/formatter"
+)
+
+const (
+	defaultOutputFormatStr     = ""
+	yamlOutputFormatStr        = "yaml"
+	yamlOutputFormatShortStr   = "y"
+	luaOutputFormatStr         = "lua"
+	luaOutputFormatShortStr    = "l"
+	tableOutputFormatStr       = "table"
+	tableOutputFormatShortStr  = "t"
+	ttableOutputFormatStr      = "ttable"
+	ttableOutputFormatShortStr = "T"
+)
+
+// OutputFormat defines a set of supported output format.
+type OutputFormat int
```

Comment:
```suggestion
// Format defines a set of supported output format.
type Format int
```
+ rename the file to `format.go`. We don't lose anything and the code will become compact.

Reply:
Done